rubocopがCIで引っかからないようにする

CIのログ(https://github.com/mightysosuke/public_photo_app/actions/runs/19462020680/job/55688214028)
```
Run bin/rubocop -f github
  bin/rubocop -f github
  shell: /usr/bin/bash -e {0}
  env:
    RUBOCOP_CACHE_ROOT: tmp/rubocop

Error: Layout/SpaceInsideArrayLiteralBrackets: Use space inside array brackets.
Error: Layout/SpaceInsideArrayLiteralBrackets: Use space inside array brackets.
Error: Process completed with exit code 1.
```

--- a/app/controllers/sessions_controller.rb
+++ b/app/controllers/sessions_controller.rb
@@ -28,7 +28,7 @@ class SessionsController < ApplicationController
       start_new_session_for user
       redirect_to after_authentication_url
     else
-      flash.now[:alert] = ["メールアドレスまたはパスワードが正しくありません。"]
+      flash.now[:alert] = [ "メールアドレスまたはパスワードが正しくありません。" ]
       render :new, status: :unprocessable_entity
     end
   end

